README.md

```diff
@@ -21,7 +21,9 @@ seen the Bash version I had before it.
 
 Many queries involve complex JQL statements, which requires authentication to execute. 
 The tool expects the property file with your user/login for OpenJDK bug database
-(comes with a glorious OpenJDK Author role, after some contributions):
+(comes with a glorious OpenJDK Author role, after some contributions).
+By default, application uses a file "auth.props" with user/pass as authenticated method.
+An empty auth.props file will switch to use anonymous authentication.
```

Comment:
I think we also need to support the case of non-existent `auth.props`, and fall back to anonymous authentication as well. This relaxes the "tool expects the property file..." sentence above as well.

Reply:
Yes, I agree, just thought that would be a different feature to handle `--auth` file, then should be in a different PR.

Reply:
No need, we can cobble all related changes in this PR.